platforms/windows/src/node.rs

```diff
@@ -703,12 +718,6 @@ macro_rules! patterns {
                 })*
             }
         }
-        $(#[allow(non_snake_case)]
-        impl paste! { [< $base_pattern_id Provider>] } {
-            $(fn $base_property_id(&self) -> Result<$com_type> {
-                self.0.resolve(|resolved| Ok(resolved.$getter().into()))
-            })*
-        })*
```

Comment:
I couldn't think of a nice way to retain the "define the simple things inside the `patterns!` macro" property given `windows` moving to traits, so for the purpose of getting it working, I inlined the simple cases into their respective traits. Technically this means that `$com_type` is now unused inside the macro.

Reply:
Please open an issue for this, then mention the issue in a comment somewhere appropriate in that file. Then we'll see if anyone can figure out an elegant solution.

Reply:
Opened #104.